spec/unit/x509/cert_provider_spec.rb

```diff
@@ -524,4 +524,30 @@ def expects_private_file(path)
       end
     end
   end
+
+  context 'CRL last update time' do
+    let(:crl_path) { tmpfile('pem_crls') }
+
+    it 'returns nil if the CRL does not exist' do
+      provider = create_provider(crlpath: '/does/not/exist')
+
+      expect(provider.crl_last_update).to be_nil
+    end
+
+    it 'returns the last update time' do
+      now = Time.now
+      Puppet::FileSystem.touch(crl_path, mtime: now)
+      provider = create_provider(crlpath: crl_path)
+
+      expect(provider.crl_last_update).to be_within(1).of(now)
+    end
+
+    it 'sets the last update time' do
+      now = Time.now
```

Comment:
Shouldn't this be a time in the past (Eg: `Time.now - 30`) to make sure that it's not calling touch without setting the mtime?